src/applications/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -103,18 +117,35 @@ class RoutedSavableReviewPage extends React.Component {
       user
     } = this.props;
 
+    const submitController = (
+      <SubmitController
+        formConfig={formConfig}
+        pageList={pageList}
+        path={path}
+        renderErrorMessage={this.renderErrorMessage}/>
+    );
+
+    let submitContent = submitController;
+
+    if (formConfig.downtime) {
+      submitContent = (
+        <DowntimeNotification
+          appTitle="application"
+          render={this.renderDowntime}
+          dependencies={formConfig.downtime.dependencies}>
+          {submitController}
```

Comment:
Why does this wrap the `submitController`?

Reply:
So it hides everything `submitController` renders?

Reply:
I think it's easier to read declarative code if the conditionals are grouped and inline. I'm trying to figure out if we can shuffle some stuff around to eliminate the need to put a partial render in a variable.

Fex rather than push a boolean into component manage the condition, just use the condition at this level to control the display. 

The back button adds some complexity to that, though. Could either have `submitController` manage that or graduate it to it's own connected component so that this layout can explicitly control when it's displayed. 

I like the latter idea because then I can see something like `showDowntimeNotification` prop and all of the pieces that are hidden / displayed because of it in one place. It's more work, though.

Reply:
We could just always render the DowntimeNotification component. I'm swapping it just to avoid another component render for forms that don't need it.

Reply:
I'm not going to mess with showing the back button. This is meant to be a simple implementation, if we want something more complicated, I think we'll need UX help.